.github/workflows/release.yaml

```
@@ -144,7 +144,16 @@ jobs:
 
             You can generate a report from this data using [devstats](https://github.com/scientific-python/devstats).
 
+            Reports for some key ecosystem packages are available at https://devstats.scientific-python.org
```

Maybe add the link under a text rather than directly?
```suggestion
            Reports for some key ecosystem packages are available at [the devstats website](https://devstats.scientific-python.org)
```

---

I thought this would be a good way to make people aware of the website URL.

---

Ahh, nevermind, this is all good, it's very unlikely to look at the logs where this appears with a screen reader or such.

---

This is for the release page description.